lib/tasks/avalon_migrations.rake

```diff
@@ -127,5 +127,33 @@ namespace :avalon do
 
       puts("Backfill complete. #{count} records updated.")
     end
+    # TODO: Test
+    desc "Migrate existing units to new Admin::Unit model and associate them with the appropriate Admin::Collections"
+    task admin_units: :environment do
+      if ENV['unit_admin_username'].nil?
+        abort "You must specify a username. This user must be part of the Unit Administrator group. Example: rake avalon:migration:admin_units unit_admin_username=user@example.edu"
+      end
+
+      units = {}
+      units_cv = Avalon::ControlledVocabulary.find_by_name('units', sort: true)
```

Comment:
Should this list be reduced to only the ones currently being used by collections?  Or is it helpful to create unit objects that are empty?